moonshot/src/configs/env_variables.py

```diff
@@ -7,6 +7,7 @@
 
 class EnvVariables(Enum):
     ATTACK_MODULES = "ATTACK_MODULES"
+    BOOKMARK = "BOOKMARK"
```

Comment:
@imda-normanchia can we have it as BOOKMARKS, jus to be inline with the other variables